openedx/features/course_rating/api/v1/serializers.py

```diff
@@ -4,22 +4,35 @@
 from rest_framework import serializers
 
 from openedx.features.course_rating.models import CourseRating, CourseAverageRating
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
 class CourseRatingSerializer(serializers.ModelSerializer):
     """
     Serializer for "CourseRating" model
     """
     course_rating = serializers.SerializerMethodField()
+    username = serializers.CharField(source='user.username', read_only=True)
+    course_title = serializers.SerializerMethodField()
 
     class Meta:
         model = CourseRating
-        fields = ['id', 'course', 'course_rating', 'rating', 'comment', 'is_approved', 'user', 'moderated_by']
+        fields = [
+            'id', 'course', 'course_title', 'username', 'course_rating', 'rating',
```

Comment:
Can you please remove the `course_rating`. The information is redundant

Reply:
course_rating contains overall course average rating and total raters and added as a requirement.